### PR TITLE
chore: remove last MPS occurence

### PR DESCRIPTION
**Pull request by RomanBredehoft**
refs https://github.com/zama-ai/concrete-ml-internal/issues/3953

### CODE REVIEW

File: docs/advanced_examples/QuantizationAwareTraining.ipynb [hunk 1]
```diff
@@ -400,11 +400,10 @@
...
    "source": [
-    "# You can also train this on GPU for example by setting\n",
-    "# MPS option is supported by macOS with Apple Silicon or AMD GPUs\n",
-    "device = (\n",
-    "    \"cuda\" if torch.cuda.is_available() else \"mps\" if torch.backends.mps.is_available() else \"cpu\"\n",
-    ")\n",
+    "# Add MPS (for macOS with Apple Silicon or AMD GPUs) support when error is fixed. For now, we\n",
+    "# observe a decrease in torch's top1 accuracy when using MPS devices\n",
+    "# FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/3953\n",
+    "device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
```
Comment (fd0r) [inline docs/advanced_examples/QuantizationAwareTraining.ipynb:406]: Should we have FIXME's in notebooks?

**Review by fd0r — APPROVED** [review]
Fine with me but a bit skeptical about having internal fixmes visible in notebooks.

File: docs/advanced_examples/QuantizationAwareTraining.ipynb (hunk 1 above)
Comment (RomanBredehoft) [reply docs/advanced_examples/QuantizationAwareTraining.ipynb:406]: well, we already have a few fixme in some notebooks, I agree it's not awesome but I think it's fine as long as it's only a few

Comment (fd0r) [reply docs/advanced_examples/QuantizationAwareTraining.ipynb:406]: alright then

**Pull request merged**